src/components/flyout/flyout_body.test.tsx

```diff
@@ -18,4 +18,13 @@ describe('EuiFlyoutBody', () => {
 
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  test('is rendered', () => {
```

Comment:
🤦 Forgot to rename this test
```suggestion
  test('scrollableTabIndex', () => {
```